### PR TITLE
Adding E2E for Framework 3.5 project

### PR DESCRIPTION
**Pull request by vagisha-nidhi**
Adding E2E test for Framework 3.5 project

### CODE REVIEW

**Comment by vagisha-nidhi** [comment]
@dotnet-bot Test TestPlatform.PR

**Review by mayankbansal018 — APPROVED** [review]

**Review by mayankbansal018 — COMMENTED** [review]
The test for framework 3.5 are compiled with old test framework, refer to CUIT E2E tests to see how we have take reference to old framework binary

**Comment by mayankbansal018** [comment]
Can we please fix this PR, & complete it?

File: test/TestAssets/SimpleTestProjectFramework35/SimpleTestProjectFramework35.csproj [hunk 1]
```diff
@@ -18,7 +18,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.VisualStudio.QualityTools">
-      <Version>$(TestPlatformExternalsVersion)</Version>
+      <Version>15.5.0-preview-1004187</Version>
```
Comment (mayankbansal018) [inline test/TestAssets/SimpleTestProjectFramework35/SimpleTestProjectFramework35.csproj]: why move to this older version?

**Pull request closed without merging**